tox.ini

```diff
@@ -1,3 +1,15 @@
+[tox]
+env_list =
+    py310
+minversion = 4.11.1
 [flake8]
 max-line-length = 230
 max-complexity = 10
+[testenv]
+description = run the tests with pytest
+package = wheel
+wheel_build_env = .pkg
+deps =
+    pytest>=6
+commands =
+    pytest test/unit_test_import_xml.py {tty:--color=yes} {posargs}
```

Comment:
We have not been using tox to test, this is a bigger change, hmm. until now it was all `nose`, which apparently is [now broken](https://github.com/nose-devs/nose/issues/1099). I _really_ wish some of these changes had come in separate PRs so we could've discussed them in isolation, but I guess we have to accept this because nose is non functional.